### ISSUE TITLE
indent split argument functions

### ISSUE THREAD

**Issue by kentcdodds** [eslint/eslint#3548]
When my function signature gets long (I realize this is an anti-pattern, but we're working on that), I get some odd indent warnings:

![screen shot 2015-08-27 at 9 03 30 am](https://cloud.githubusercontent.com/assets/1500684/9524268/4c19f9a4-4c9b-11e5-9ee0-05c81f7f5c03.png)

Here's the code (for copy/paste):

**index.js** (this is how lots of my code looks like today)

```javascript
function foo(bar, baz,
             eggs, spam) {
  bar(eggs, baz, spam);
}

foo();
```

**index.passes.js**

```javascript
function foo(bar, baz,
             eggs, spam) {
               bar(eggs, baz, spam); // <-- this is kinda crazy
             } // <-- crazy

foo();
```

**.eslintrc**

```
{
  "rules": {
    "indent": [2, 2]
  }
}
```

Is there anything I can do to tell ESLint to not care about the fact that the arguments are on multiple lines? Or could I somehow have my arguments on multiple lines in a different way to make it so I still get the indentation of 2?

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by BYK** [comment]
This should be fixed with #3467 - just not released yet.

**Comment by kentcdodds** [comment]
Found [an answer](https://twitter.com/VilleImmonen/status/636917524521701376):

```javascript
function foo(bar, baz,
             eggs, spam
) {
  bar(eggs, baz, spam);
}

foo();
```

This is sufficient... Now I just need to refactor a bunch of code :-)

**Comment by BYK** [comment]
>Now I just need to refactor a bunch of code :-)

Or you can wait for the upcoming release ;)

**Comment by kentcdodds** [comment]
Whoops, race condition with you @BYK. Thanks! Any word on when that next release will be? (As an open source author myself, I'm sorry to be asking that question).

**Comment by BYK** [comment]
@kentcdodds - Heh, no problem. Release is a @nzakas question though :)
